stable/datadog/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
   - name: kube-state-metrics
-    version: ~0.9.0
+    version: "0.11.0"
```

Comment:
Let's keep the patch-level version match as described in https://docs.helm.sh/chart_best_practices/#versions